judge/forms.py

```diff
@@ -78,9 +80,9 @@ def clean(self):
         max_orgs = settings.DMOJ_USER_MAX_ORGANIZATION_COUNT
 
         if sum(org.is_open for org in organizations) > max_orgs:
-            raise ValidationError(ngettext('You may not be part of more than {count} public organization.',
-                                           'You may not be part of more than {count} public organizations.',
-                                           max_orgs).format(count=max_orgs))
+            raise ValidationError(ngettext_lazy('You may not be part of more than {count} public organization.',
```

Comment:
What's the point of doing this lazily?

Reply:
To avoid an extra import

Reply:
In theory, imports are a lot cheaper than doing the extra work every invocation with laziness, but whatever.